packages/core/src/components/Swiper/SwiperItem.tsx

```diff
@@ -0,0 +1,30 @@
+import { COMPONENT_TYPES } from "@draftbit/types";
+import React from "react";
+import { View, StyleSheet, StyleProp, ViewStyle } from "react-native";
+
+const styles = StyleSheet.create({
+  wrapper: {
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "center",
+  },
```

Comment:
Should these be user-configurable? If so we should put them in the `layout` object in `SEED_DATA`. If not, disregard

Reply:
Actually, It is not required here, moved it to the example.